images/win/scripts/Installers/Install-DebianImage.ps1

```diff
@@ -0,0 +1,29 @@
+################################################################################
+##  File:  Install-DebianImage.ps1
+##  Desc:  Install Debian Image for WSL
+################################################################################
+
+Import-Module -Name ImageHelpers
+
+debianTempDir = "C:\Temp\debian"
```

Comment:
Is it possible to do not hardcode path to the temp folder?

Reply:
Sure, I changed the path to `"$env:AGENT_TEMPDIRECTORY\debian"`